app/views/activities/index.html.slim

```diff
@@ -11,9 +11,9 @@ div.row
           label.radio
             = radio_button_tag :kind, kind == 'all' ? '' : kind, params[:kind] == kind
             = " " + kind == 'feed_entry' ? 'Blog Post' : kind.titleize
-        label.team_filter
-          ' Team:
-          = select_tag :team_id, options_for_select(teams.map { |t| [t.display_name, t.id] }, params[:team_id]), include_blank: true, class: 'form-control'
+        - if teams.count != 0
```

Comment:
Because of readabilty, I'd vote for:
- `unless teams.count = 0` 
or
- `if teams.any?`

Reply:
Yes I again make a pull request after changing it to 
```
if teams.any?
```
Thanks for the suggestion.